ui/rdb/views/CommandsView.svelte

```diff
@@ -0,0 +1,121 @@
+<script lang="ts">
+    window.vscode = acquireVsCodeApi();
+    import {odc} from "../ExtensionIntermediary";
+    import { commandsView } from "./CommandsView";
+
+    const storage = window.localStorage;
```

Comment:
Is there any type of cleanup we need to do with localStorage on a fresh launch of the app so we don't end up in an inconsistent/invalid state?

Reply:
We're mostly just storing what they put in to simplify the case where they want to do the same commands over and over across launches so currently we wouldn't want to remove or cleanup any of that since it's there for that reason